aten/src/ATen/native/sparse/SparseCsrTensorMath.cpp

```diff
@@ -90,7 +90,7 @@ Tensor& unary_op_out(F op_out, const Tensor& self, Tensor& result) {
   auto self_values = self.values();
   auto result_values = result.values();
 
-  op_out(self_values, result_values);
+  op_out(self_values, std::forward<Args>(args)..., result_values);
```

Comment:
Why the use of `std::forward` here? `std::forward` is used to do perfect forwarding when having things of the form `Args&&` for `Args`  a template. It's reasonable to write that here, but then you're missing the perfect forwarding ampersands.

Reply:
I wasn't aware of this, thanks @Lezcano - should be fixed now. :)